metricbeat/docs/fields.asciidoc

```diff
@@ -6728,7 +6728,7 @@ The current number of idle client connections waiting for a request.
 
 
 [[exported-fields-php_fpm]]
-== php_fpm Fields
+== PHP-FPM Fields
```

Comment:
I would say to leave the name of the module as it shows in the configuration (with `_` instead of `-`) as otherwise might be confusing for the user.